views/signingreturn.hbs

```diff
@@ -146,17 +146,17 @@
 					<p>
 						<strong>ttl_expired</strong>
 						<br />
-						The token was not used within the timeout period or the token was already accessed. 
+						The token was not used within the timeout period or the token was already accessed.
 					</p>
 
 					<p>
 						<strong>viewing_complete</strong>
 						<br />
-						The recipient did not need to sign but has completed the viewing ceremony. 
+						The recipient did not need to sign but has completed the viewing ceremony.
 					</p>
 
 					<p>
-						<a href="https://developers.docusign.com/esign-rest-api/guides/features/embedding" target="_blank">Learn more about Embedded Signing</a>
+						<a href="https://developers.docusign.com/docs/esign-rest-api/esign101/concepts/embedding/#focused-view" target="_blank">Learn more about Focused view</a>
```

Comment:
Can we be consistent about capitalization? I think (ask Matt) that it's lower case... you had it lower case earlier, at least let's be consistent ...